src/js/port_handler.js

```diff
@@ -82,13 +77,16 @@ PortHandler.initialize = function () {
                 tcpPorts.forEach(function (port) {
                     const removePort = () => {
                         self.mdnsBrowser.services = self.mdnsBrowser.services.filter(s => s.fqdn !== port.fqdn);
+                        self.mdnsBrowser.browser.services=[];
```

Comment:
@ASDosjani you might want to check with 2 eLRS RX connected and check this doesn't mess with the other eLRS port when one is disconnected.